sqladmin/auth/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Boolean, Column, Integer, String
+from sqlalchemy.ext.declarative import declarative_base
+
+from sqladmin.auth.hashers import make_password, verify_password
+
+Base = declarative_base()
+
+
+class User(Base):  # type: ignore
```

Comment:
Actually I thought we would integrate with something like https://www.starlette.io/authentication/ . I guess it would be easier to leave these out of the admin itself. what do you think?

Reply:
In fact, I'm more familiar with FastAPI.

Your proposal is very good, I will look at the link you sent tomorrow and try to modify it.

Reply:
I think they should basically be the same. Sure.